src/bin/common/parse_command.ml

```diff
@@ -361,8 +361,8 @@ let mk_limit_opt age_bound fm_cross_limit timelimit_interpretation
     `Ok()
 
 let mk_output_opt
-    interpretation use_underscore unsat_core output_format model_type
-    () () () ()
+    interpretation use_underscore objectives_in_interpretation unsat_core
+    output_format model_type () () () ()
```

Comment:
The line isn't relevant. I don't want to remove it. We should add an extra case for the `get-info` statement to print the objectives on the diagnostic channel and keep this CLI option in case we use the native input language.

Reply:
I open a remainder issue #891